build.sbt

```diff
@@ -4,9 +4,8 @@ scalaVersion in ThisBuild := "2.11.12"
 crossScalaVersions in ThisBuild := Seq("2.11.12", "2.12.12", "2.13.3")
 //scalaJSUseRhino in ThisBuild := true
 
-lazy val scalatest = "org.scalatest" %% "scalatest" % "3.2.2"
-lazy val specs2 = "org.specs2" %% "specs2-core" % "4.10.5"
-lazy val scalameta = "org.scalameta" %% "scalameta" % "4.3.24"
+lazy val scalatest = Def.setting("org.scalatest" %%% "scalatest" % "3.2.2")
```

Comment:
could you explain this notation to me? Is this relevant to scala.js?
the standard dependency declaration gets automatically updated by a bot, i'm not sure if it's compatible with this form

Reply:
Sure. You might already know how `%%` adds a Scala version suffix, which enables cross-building between Scala versions. The third `%` sign introduces another cross-building axis, by adding the platform part (e.g. `_sjs1`) to the artifact name, so the JS project JAR dependencies are downloaded. As can be seen in Scalamock, the JS version of the same class can differ, also the Scala.js JARs contain additional intermediate code representation and may contain JS dependencies metadata. This mechanism is actually a part of `sbt-crossproject` and works for Scala Native as well.

Due to the lack of SJS versions of ScalaTest and Specs2, no test suites were discovered in `scalamockJS/test`, so the CI was only checking whether ScalaMock compiles and links.

I believe scala-steward can handle this notation, we're using it in https://github.com/UdashFramework/udash-core where the version numbers are even harder to find, as they're extracted to a constant in another file.

Reply:
:+1: